api/controllers/StatsController.js

```diff
@@ -6,6 +6,40 @@ dayjs.extend(isBetween);
 /////////////
 // Helpers //
 /////////////
+/**
+ * Find the season in which a given timestamp occurred using binary search
+ * @param {*} sortedSeasons - Seasons sorted (OLDEST FIRST)
```

Comment:
So it feels kinda strange to me to require that the seasons be sorted in DESC chronological order (oldest first), but that's what the api has been delivering in the end and what the frontend expects in order to put the oldest/current season in the top of the drop-down list. 

I initially had implemented this function so that it expected the seasons to be sorted in ASC chronological order ([commit](https://github.com/cuttle-cards/cuttle/commit/ded2b2195a4362b970e18fee2c049128ee746ebb)), but that meant that we had to loop again to transform the array back to DESC order before displaying it (in that commit I did it clientside in the page's `created()` hook), which just seems unnecessary given that the only reason to sort it ASC was because that seemed more "natural" for a binary search. 

But given that this function is only called in this controller I'm a context where were ultimately going to deliver the seasons in DESC chronological order, I think it actually reads better this way and also thereby skips the need to flip the sort order back and forth. It just has the quirks that the `sortedSeasons` param needs to be sorted in DESC chronological order (which it already is when returned by the `getSeasonsWithoutRankings()` helper and that the higher/lower cases below decrement `maxIndex` and increment `minIndex` respectively, rather than the other way around.